README.md

```diff
@@ -1,32 +1,25 @@
 # KerasNLP: Modular NLP Workflows for Keras
 [![](https://github.com/keras-team/keras-nlp/workflows/Tests/badge.svg?branch=master)](https://github.com/keras-team/keras-nlp/actions?query=workflow%3ATests+branch%3Amaster)
-![Python](https://img.shields.io/badge/python-v3.8.0+-success.svg)
-![Tensorflow](https://img.shields.io/badge/tensorflow-v2.5.0+-success.svg)
+![Python](https://img.shields.io/badge/python-v3.9.0+-success.svg)
 [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/keras-team/keras-nlp/issues)
 
-KerasNLP is a natural language processing library that works natively 
-with TensorFlow, JAX, or PyTorch. Built on [Keras Core](https://keras.io/keras_core/announcement/),
-these models, layers, metrics, callbacks, etc., can be trained and serialized
-in any framework and re-used in another without costly migrations. See "Using 
-KerasNLP with Keras Core" below for more details on multi-framework KerasNLP.
-
-KerasNLP supports users through their entire development cycle. Our workflows 
-are built from modular components that have state-of-the-art preset weights and 
-architectures when used out-of-the-box and are easily customizable when more 
-control is needed.
-
-This library is an extension of the core Keras API; all high-level modules are 
-[`Layers`](https://keras.io/api/layers/) or 
-[`Models`](https://keras.io/api/models/) that receive that same level of polish 
-as core Keras. If you are familiar with Keras, congratulations! You already 
-understand most of KerasNLP.
+KerasNLP is a natural language processing library that works natively
```

Comment:
I removed trailing spaces, but I think that is making this diff look a lot bigger than it is.

Reply:
Yeah, there's something crazy going on with the diff here. Even identical paragraphs are marked as deleted.